training/train_vit_vqvae.py

```diff
@@ -376,7 +406,7 @@ def compute_loss(params):
     train_time = 0
     epochs = tqdm(range(num_epochs), desc=f"Epoch ... (1/{num_epochs})", position=0)
 
-    wandb.init(config=parser.parse_args())
+    wandb.init(config=flat_args(model_args, data_args, training_args))
```

Comment:
Oh there was an issue with previous method?

Reply:
Only when supplying the arguments using a json file rather than command line arguments, see https://github.com/patil-suraj/vit-vqgan/blob/main/training/train_vit_vqvae.py#L203. In that case the parser stops the program because it thinks the required arguments have not been supplied.